lib/utility/logger.rb

```diff
@@ -43,6 +44,14 @@ def error_with_backtrace(message: nil, exception: nil, prog_name: nil)
       def new_line
         logger.info("\n")
       end
+
+      def generate_trace_id
+        SecureRandom.uuid
+      end
+
+      def abbreviated_message(message)
+        message.gsub(/[\n\t]/, ' ').gsub(/\s+/, ' ')[0, MAX_SHORT_MESSAGE_LENGTH].strip
+      end
```

Comment:
We can probably use `https://apidock.com/rails/String/truncate` here - but what are gsubs here for? 
I see first one replaces `\n\t` - I think we can remove it from errors that we raise.

Reply:
We don't have `rails` here, do we? As to the gsubs, - we're supposed to clean up line breaks from messages that we bubble up, but it doesn't mean that we need to remove the breaks from the logs - because in the logs, the breaks are for improving the readability. I don't like the idea of just getting rid of all the breaks that make the long text blocks structured.

Reply:
It's available within `active_support/core_ext/string/filters` - https://www.rubydoc.info/gems/activesupport/String:truncate

My point here is that we wanna display user-friendly error messages, that means that we need to actually sort out the errors ourselves and never show messages of errors that we do not control (e.g. errors raised by ruby/used libraries and anything). In this case we won't need to replace `\n\t` and such, our error handling layer will wrap these errors in user-friendly errors while also logging the real ones, but this indeed is something that we can handle in future.

Reply:
@artem-shelkovnikov got it. Changing the code.